roles/kubernetes/preinstall/tasks/0050-create_directories.yml

```diff
@@ -19,11 +19,21 @@
     - node
   with_items:
     - "{{ kube_config_dir }}"
-    - "{{ kube_cert_dir }}"
     - "{{ kube_manifest_dir }}"
     - "{{ kube_script_dir }}"
     - "{{ kubelet_flexvolumes_plugins_dir }}"
 
+- name: Create kubernetes cert directories
```

Comment:
Hmm I think the "Create other directories" task just below also do that, maybe it should be created there?

Reply:
That is a nice point.
We just need to move `{{ kube_cert_dir }}` into the existing `Create other directories` task.
It is better to rename the `Create other directories` task to `Create other directories of root owner` or something to clarify what is different from `Create kubernetes directories` task.